lib/mapper.js

```diff
@@ -92,3 +92,24 @@ exports.page = function(msg){
     data: convert(props, time)
   }
 };
+
+/**
+ * Map screen `msg`.
+ *
+ * @param {Facade} msg
+ * @param {Object} settings
+ * @return {Object}
+ */
+
+exports.screen = function(msg){
+  var props = traverse(msg.properties());
+  var name = `Viewed ${msg.name()} Screen`;
+  delete props.name;
+
+  return {
+    timestamp: time(msg.timestamp()),
+    type: 'track',
```

Comment:
@pfeiffer is this what we want to set here? Not sure amplitude has a "screen" counter type for `type` here given that we are sending `'page'` for `.page()` calls.